Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,180 @@
+var chosenWord = randomWord ?? "" //unpacks the word
+var arrayChosenLetters = Array(chosenWord) // changes random word into an array, word chosen by computer
+var letterInputArray = Array<String>() //place for the letters inputed by player
+var triesLeft = 6 //The user must have a limited number of guesses
+var guess = 0 // times the player tried
+var underScore: [Character] = Array(String(repeating: "_", count: chosenWord.count))
+let alphabet = ["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"]
+let filter = alphabet
+
+
+var asciiLost = """
+ ____________
+    |/      |
+    |      (_)
+    |       |
+    |       |
+    |      | |
+    |
+    |
+  __|___
+"""
+
+var ascii1 = """
+ __________
+    |/    
+    |
+    |
+    |
+    |
+    |
+  __|___
+"""
+
+var ascii2 = """
+ ____________
+    |/      |
+    |
+    |
+    |
+    |
+    |
+  __|___
+"""
+
+var ascii3 = """
+ ____________
+    |/      |
+    |      (_)
+    |
+    |
+    |
+    |
+  __|___
+"""
+
+var ascii4 = """
+ ____________
+    |/      |
+    |      (_)
+    |       |
+    |
+    |
+    |
+  __|___
+"""
+
+var ascii5 = """
+ ____________
+    |/      |
+    |      (_)
+    |       |
+    |       |
+    |
+    |
+  __|__
+"""
+
+var ascii6 = """
+ ____________
+    |/      |
+    |      (_)
+    |       |
+    |       |
+    |      |
+    |
+    |
+  __|___
+"""
+
+print(ascii1)
+print("Choose one letter and hit enter")
+
+hangman: repeat {
+
+    print(String(underScore))
+//    print(chosenWord)
+    
+    let letterInput = readLine()?.lowercased() ?? "0"
+    
+    if letterInput.count > 1 || !filter.contains(letterInput){
+        print("Please enter one letter")
+        
+        continue
+    }
+       switch triesLeft {
+       case 6:
+           print(ascii2)
+       case 4:
+           print(ascii3)
+       case 3:
+           print(ascii4)
+       case 2:
+           print(ascii6)
+       case 1:
+           print(ascii5)
+//       case 0:
+//           print(asciiLost)
+//           print("Nice try, you lost. the word was \(chosenWord)  Please play again")
+       default:
+           print(ascii2)
+    }
+    if !chosenWord.contains(letterInput) && triesLeft != 0 {
+        triesLeft -= 1
+        guess += 1
+        letterInputArray.append(letterInput)
+        let printTriesLeft = String(triesLeft)
+           print("You typed in the wrong letter \(letterInput), you have \(printTriesLeft) attempts left. You already tried \(letterInputArray)")
+        continue
+    }
+    for (index, char) in chosenWord.enumerated() where chosenWord.contains(letterInput) {
+        if char == Character(letterInput) {
+            underScore[index] = Character(letterInput)
+            guess += 1
+        }
+    
+    if !underScore.contains("_") {
+        print("You won!!!! it took \(guess) guesses to find the word \(chosenWord)")
+        break hangman
```

Comment:
This code breaks out of your loop, remember that the user should be able to replay the game after they have won or lost when the requirement mentions that it should end the current game, it means that it should reset the game but I understand where there may be some confusion.